modules/book.js

```diff
@@ -0,0 +1,61 @@
+import initialBooks from './data.js';
+
```

Comment:
- [ ] I am able to access the `Book List` page, thus unable to tell if the added books are being displayed to the page or not, please kindly fix the issue, ensure the link to the `Book List` page is working so that we can see the added books displayed on the page.